ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/StatementExecutorTest.java

```diff
@@ -449,6 +451,37 @@ public void shouldEnforceReferentialIntegrity() {
 
   }
 
+  @Test
+  public void shouldFailIfExceedActivePersistentQueriesLimit() {
```

Comment:
We should have a similar test for INSERT INTO

Reply:
Also add a test to make sure the persisted config is used.

Reply:
Yes you are correct - it's already validating that the persisted config is used for the limit check. So never mind my comment :)